.sailsrc

```diff
@@ -1,5 +1,9 @@
 {
   "generators": {
     "modules": {}
+  },
+  "hooks": {
```

Comment:
Put this is by mistake, want to test this functionality separately. I'll remove it